india_compliance/gst_india/api/__init__.py

```diff
@@ -0,0 +1,32 @@
+import json
+
+import frappe
+from frappe.utils.password import get_decrypted_password
+
+
+@frappe.whitelist()
+def get_gst_api_secret():
+    frappe.only_for("System Manager")
+    return get_decrypted_password(
+        "GST Settings", "GST Settings", "api_secret", raise_exception=False
+    )
+
+
+@frappe.whitelist()
+def set_gst_api_secret(api_secret):
+    frappe.only_for("System Manager")
+    frappe.set_value("GST Settings", None, "api_secret", api_secret)
```

Comment:
`db.set_value`? I don't believe `frappe.client.set_value` was intended (this is what is aliased by `frappe.set_value`)